frontend/src/App.tsx

```diff
@@ -133,6 +134,12 @@ const App = (): React.ReactElement => {
                       component={PasswordResetSuccessPage}
                     />
 
+                    <Route
```

Comment:
Close, this should be a private route!
```suggestion
                    <PrivateRoute
```